### PR TITLE
build: fix a build failure related to conditional compilation

### PR DESCRIPTION
**Pull request by jiangliu**
Fix a build failure related to conditional compilation: error[E0063]: missing field `id_mapping` in initializer of `VfsOptions`
   --> src/api/vfs/mod.rs:751:16
    |
751 |             Ok(VfsOptions {
    |                ^^^^^^^^^^ missing `id_mapping`

### CODE REVIEW

File: src/api/vfs/mod.rs [hunk 1]
```diff
@@ -723,44 +723,70 @@ pub mod persist {
...
-        no_opendir: bool,
-        no_writeback: bool,
         in_opts: u64,
         out_opts: u64,
-        killpriv_v2: bool,
         no_readdir: bool,
         seal_size: bool,
+        id_mapping_internal: u32,
+        id_mapping_external: u32,
+        id_mapping_range: u32,
```
Comment (bergwolf) [inline src/api/vfs/mod.rs:732]: why introducing new fields in a clean up PR?

**Comment by bergwolf** [comment]
How do you reproduce it? I don't see it locally, nor does the CI catch it.

File: src/api/vfs/mod.rs (hunk 1 above)
Comment (jiangliu) [reply src/api/vfs/mod.rs:732]: It's not a clean up, but a bugfix:)

**Comment by jiangliu** [comment]
> How do you reproduce it? I don't see it locally, nor does the CI catch it.

Can be reproduced by `cargo build --all-features`. Will enhance the makefile to cover feature `persist`.

**Review by bergwolf — APPROVED** [review]
lgtm, thanks!

**Pull request merged**